tests/test_datasets.py

```diff
@@ -113,6 +114,30 @@ def test_factory(env_name, repo_id, policy_name):
             assert key in item, f"{key}"
 
 
+def test_multilerobotdataset_frames():
+    """Check that all dataset frames are incorporated."""
+    # Note: use the image variants of the dataset to make the test approx 3x faster.
+    repo_ids = ["lerobot/aloha_sim_insertion_human_image", "lerobot/aloha_sim_transfer_cube_human_image"]
+    sub_datasets = [LeRobotDataset(repo_id, root="tests/data") for repo_id in repo_ids]
+    dataset = MultiLeRobotDataset(repo_ids, root="tests/data")
```

Comment:
We dont use root for now. We might when we change the API (DATA_DIR="tests/data" is ugly)
```suggestion
    sub_datasets = [LeRobotDataset(repo_id) for repo_id in repo_ids]
    dataset = MultiLeRobotDataset(repo_ids)
```